lib/widgets/app.dart

```diff
@@ -268,41 +266,50 @@ class ChooseAccountPage extends StatelessWidget {
         // The default trailing padding with M3 is 24px. Decrease by 12 because
         // IconButton (the "…" button) comes with 12px padding on all sides.
         contentPadding: const EdgeInsetsDirectional.only(start: 16, end: 12),
-        onTap: () => Navigator.push(context,
-          HomePage.buildRoute(accountId: accountId))));
+        onTap: () => HomePage.navigate(context, accountId: accountId)));
   }
 
   @override
   Widget build(BuildContext context) {
+    final colorScheme = ColorScheme.of(context);
```

Comment:
This commit also edits colors for the individual `ListTile`s above:
```diff
       child: ListTile(
         title: title,
         subtitle: subtitle,
+        tileColor: colorScheme.secondaryContainer,
+        textColor: colorScheme.onSecondaryContainer,
```

but then the main commit reverts that.

It doesn't look like the main commit is making some other change that compensates for removing it, so that change probably doesn't belong in the main commit. Are those lines not actually needed in the first place?

Reply:
Those lines should be kept to fix the appearance of the account cards on choose-acocunt page.